src/main/resources/application-local.yml

```diff
@@ -51,12 +48,12 @@ no.nav.security.jwt:
     registration:
       ef-sak-client-credentials:
         resource-url: ${familie.ef.sak.url}
-        token-endpoint-url: http://localhost:8092/mockapi/token
+        token-endpoint-url: https://login.microsoftonline.com/navq.onmicrosoft.com/oauth2/v2.0/token
```

Comment:
Syns lokal test burde virke utenom avhengigheter til tjenester utenom det som kjører lokalt. Dette medfører att man må ha nett for att det skal virke

Reply:
I hvilke tilfeller jobber du helt uten nett? 🤔  Hvis du jobber med sak lokalt så trenger du nett uansett fordi den krever preprod-autentisering. Regner med at man ikke jobber med mottak-sak-kommunikasjon uten å jobbe med sak samtidig.

Reply:
Det er mulig nav-testmiljø/microsoft alltid er oppe. Jeg har iaf jobbet i prosjekt hvor testmiljøn ikke alltid er like stabil eller at nettet detter ned, og av den grunnen er det fint hvis man fortsatt kan jobbe utenom avhengigheter til eksterne avhengigheter lokalt.

Reply:
Jeg har i hvert fall aldri opplevd problemer mot microsoft. Jeg mener behovet for å jobbe med ordentlige saksbehandler-brukere og ha lokalt miljø så likt testmiljø som mulig trumfer behovet for å jobbe uten nett. Men kanskje det er flere som har meninger om dette? 😄